keep-ui/entities/workflows/model/schema.ts

```diff
@@ -33,6 +33,7 @@ export const V2StepAlertTriggerSchema = z.object({
   properties: z.object({
     alert: AlertTriggerValueSchema,
     source: z.string().optional(),
```

Comment:
Shouldn't it be removed now? only cel? or do we support backward compatibility?

Reply:
we have backward compatibility. It's only for the purpose of validation, so having `source` will work as expected